spec/js_routes/rails_routes_compatibility_spec.rb

```diff
@@ -14,15 +14,15 @@
     expect(evaljs("Routes.inbox_path(1)")).to eq(test_routes.inbox_path(1))
   end
 
-  it "should raise error if required argument is not passed" do
+  it "should raise error if required argument is not passed", :aggregate_failures do
     expect { evaljs("Routes.thing_path()") }
-      .to raise_error('Route parameter missing: id')
+      .to raise_error(/parameter missing: id/)
```

Comment:
Why is it changed?

Reply:
Because mini_racer return `Error: Route parameter missing: id` as error message. But for jruby therubyrhino gem will return message `Route parameter missing: id`

Reply:
Made `/Route parameter missing .. /` regex